multirun/multirun.go

```diff
@@ -83,10 +87,11 @@ func (m multirun) boundedExecution(ctx context.Context) error {
 	for w := 0; w < m.jobs; w++ {
 		go func() {
 			defer wg.Done()
-			err := m.spawnWorker(ctx, commands)
+			err := m.spawnWorker(ctx, commands, errs)
```

Comment:
I suggest to change the function to return a `bool`. `true` if it stopped because of an error. The code can be cleaned up here and double-if can be removed to be `if errorOccurred && m.stopOnError`.